app/server-control/directives/kvm-console.js

```diff
@@ -30,11 +30,18 @@ window.angular && (function(angular) {
                 return false;
               };
 
+              var flag = 0;
```

Comment:
Can you use a better variable name than flag?

Reply:
Okay, I will use "KVMconn" to replace it.